easybuild/tools/toolchain/mpi.py

```diff
@@ -188,10 +188,16 @@ def mpi_cmd_for(self, cmd, nr_ranks):
         # Intel MPI mpirun needs more work
         if mpi_family == toolchain.INTELMPI:  # @UndefinedVariable
 
-            tmpdir = tempfile.mkdtemp(prefix='eb-mpi_cmd_for-')
-
             # set temporary dir for mdp
-            env.setvar('I_MPI_MPD_TMPDIR', tmpdir)
+            # note: this needs to be kept *short*, to avoid mpirun failing with "socket.error: AF_UNIX path too long"
+            # exact limit is unknown, but ~20 characters seems to be OK
```

Comment:
can't include it without breaking the column limit :package: 

it's mentioned in this PR, and it turns up pretty quickly when googling `AF_UNIX path too long` mentioned here, so should be fine without it